docs/core/integration-quality-scale/rules/iqs048.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "IQS048 - The documentation described the provided service actions that can be used"
+---
+
+IQS048 is as follows:
+
+The documentation described the provided service actions that can be used
+
+## Reasoning
+
+Integrations can register service actions to provide functionality that is not possible with normal entities.
+These service actions can be harder to use, so we want to make sure that the documentation describes both what they do, and what the parameters are.
+
+## Example implementation
+
+```markdown
+## Actions
+
+The integration provides the following actions.
+
+### Get schedule
+
+The `my_integration.get_schedule` service is used to fetch a schedule from the integration.
+
+| Data attribute         | Optional | Description                                          |
+|------------------------|----------|------------------------------------------------------|
+| `config_entry_id`      | No       | The ID of the config entry to get the schedule from. |
```

Comment:
I do know that I heard that tables are meh and that we need a better way to display them, what do we want? Because this is probably the best way to get integrations to use the new format

Reply:
I don't think we need to decide the presentation now. We can change that later when we know a better alternative.

Reply:
I agree with Martin on this one.

While we do have the intention to change it, we are not there yet. So let's not.